src/js/components/Collapsible/Collapsible.js

```diff
@@ -77,7 +77,7 @@ const Collapsible = forwardRef(
           collapsible: { minSpeed, baseline },
         } = theme;
         const nextSpeed = Math.max(
-          (sizeRef.current / baseline) * minSpeed,
+          ((sizeRef.current || 0) / baseline) * minSpeed,
```

Comment:
I think a cleaner way to do this would be to adjust the useRef to be `useRef(0)`, that way it starts with an initial value.